packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -23,6 +22,11 @@ export interface Props extends HTMLAttributes<HTMLDivElement> {
   buttonContent?: React.ReactNode | string;
   bottomSpacing?: number;
   topSpacing?: number;
+  // The role property of the alert component (e.g. 'log', 'status' or 'alert'.) Defaults to 'alert'.
+  // More info: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#roles_with_implicit_live_region_attributes
+  role?: AriaRole;
```

Comment:
wonder whether we should just infer this role from the `severity` instead of exposing it as a prop? 🤔

Reply:
Why not both? if no role then calculate it based on the severity.

Reply:
Aaa, nice, I completely missed that it is extending the `HTMLAttributes<HTMLDivElement>`!

I think what I will do then is that if the role is not defined manually then it will be inferred based on the `severity`!